packages/datadog-instrumentations/src/bluebird.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+const { addHook } = require('./helpers/instrument')
+const { wrapThen } = require('./helpers/promise')
+const shimmer = require('../../datadog-shimmer')
+
+const DD_LIB_COPIES = '_datadog_library_copies'
+
+function createGetNewLibraryCopyWrap (originalLib) {
+  return function wrapGetNewLibraryCopy (getNewLibraryCopy) {
+    return function getNewLibraryCopyWithTrace () {
+      const libraryCopy = getNewLibraryCopy.apply(this, arguments)
+      shimmer.wrap(libraryCopy.prototype, '_then', wrapThen)
+      shimmer.wrap(libraryCopy, 'getNewLibraryCopy', createGetNewLibraryCopyWrap(originalLib))
+      addToLibraryCopies(originalLib, libraryCopy)
+      return libraryCopy
+    }
+  }
+}
+
+function addToLibraryCopies (originalLib, libraryCopy) {
```

Comment:
Library copies were used for unpatching support and should no longer be needed.

Reply:
done